### DISCUSSION TITLE
Empty Back Office page after install version 4.2.x on DNN 9.13

### DISCUSSION THREAD

**Discussion by hienccn** [openstore-ecommerce/OpenStore#303] [General]
I have setup new DNN 9.13.4 and installed OpenStore 4.2.2 following the guide (https://doc.openstore-ecommerce.com/), after applying OpenStoreBO to the OpenStore Admin page, it does not display anything. I have tried many times, including older versions of DNN but cannot access the BO page.

Does version 4.2.2 support the latest version of DNN?

**Comment by leedavi** [comment]
Yes, it does.  Did the back office exist before the upgrade and then was just empty after the upgrade?
